scripts/start_manager.sh

```diff
@@ -15,7 +15,7 @@ fi
 #    exit 1
 # fi
 
-cd "${SQL_COMPILER_DIR}" && ./build.sh
+#cd "${SQL_COMPILER_DIR}" && ./build.sh
```

Comment:
we don't do this anymore (I'm not using this script so I don't mind)

Reply:
this is committed by mistake